sources/utils/Animations.swift

```diff
@@ -0,0 +1,22 @@
+//  Copyright 2018, Oath Inc.
+//  Licensed under the terms of the MIT License. See LICENSE.md file in project root for terms.
+
+import Foundation
+
+extension UIImageView {
+    func enableRotation() {
```

Comment:
We already have same code in `source/Utils` =\

Reply:
So? I don't want to make them `public` for everyone.